expenses/reports.py

```diff
@@ -241,6 +267,16 @@ def get_column_headers(self, engine: Engine) -> (typing.List[str], typing.List[s
             return ([_("Category"), _("Total")], ["left", "right"])
         elif self.query_type == "month":
             return ([_("Month"), _("Total")], ["right", "right"])
+    
+    def get_column_header_names(self, engine: Engine) -> (typing.List[str], typing.List[str]):
```

Comment:
I’d prefer to add a `is_html=True` parameter to `get_column_headers` instead of duplicating code. The addition of a method would also need to be reflected in other reports (VendorStats can’t be rendered as CSV with this code, for example).